src/adapter/src/coord/introspection.rs

```diff
@@ -160,11 +160,10 @@ pub fn auto_run_on_introspection<'a, 's, 'p>(
     if (has_dependencies && valid_dependencies)
         || (!has_dependencies && !could_run_expensive_function)
     {
-        let intros_cluster = catalog.resolve_builtin_cluster(&MZ_INTROSPECTION_CLUSTER);
-        tracing::debug!("Running on '{}' cluster", MZ_INTROSPECTION_CLUSTER.name);
+        tracing::info!("Running on '{}' cluster", MZ_INTROSPECTION_CLUSTER.name);
 
         // If we're running on a different cluster than the active one, notify the user.
-        if intros_cluster.name != session.vars().cluster() {
```

Comment:
Why did this change? Is there a guarantee that a `BulitinCluster.name` is always the same as a `ClusterReplica.name`?

Reply:
Made the change because it seemed simpler. intros_cluster is a Cluster but you're right that there's no guarantee that BuiltinCluster.name is the same as Cluster.name although in this case they are. Do you want me to revert this change?

Reply:
Yeah, revert please.